plugin/log/README.md

```diff
@@ -10,14 +10,10 @@ log
 
 * With no arguments, a query log entry is written to *stdout* in the common log format for all requests
 
-~~~ txt
-log [stdout]
-~~~
-
```

Comment:
I would leave this in, but drop the `[stdout]` bit

Reply:
Right but then the statement becomes the same as is on line 7-9...

Reply:
/me checks out PR. Ah yes, you're richt.

Reply:
=)